backend-beeyond/src/main/java/at/htl/beeyond/dto/UserListDto.kt

```diff
@@ -0,0 +1,17 @@
+package at.htl.beeyond.dto
+
+import at.htl.beeyond.entity.User
+import at.htl.beeyond.validation.Checks
+import at.htl.beeyond.validation.Exists
+import javax.validation.GroupSequence
+
+@GroupSequence(value = [Checks.UserListExists::class, UserListDto::class])
+data class UserListDto(
+    @field:Exists(
+        entity = User::class,
+        fieldName = "name",
+        groups = [Checks.UserListExists::class]
```

Comment:
you don't need this line. You can check everything from `UserListDto`.
So delete this groups and change the `@GroupSequence` to `@GroupSequence(value = [UserListDto::class])`